checks.go

```diff
@@ -98,7 +99,12 @@ func (m *Sealing) checkCommit(ctx context.Context, si SectorInfo, proof []byte,
 		return &ErrBadSeed{xerrors.Errorf("seed epoch doesn't match on chain info: %d != %d", pci.PreCommitEpoch+miner.PreCommitChallengeDelay, si.SeedEpoch)}
 	}
 
-	seed, err := m.api.ChainGetRandomness(ctx, tok, crypto.DomainSeparationTag_InteractiveSealChallengeSeed, si.SeedEpoch, nil)
+	buf := new(bytes.Buffer)
+	if err := m.maddr.MarshalCBOR(buf); err != nil {
```

Comment:
@magik6k - What impact does passing the `nil` value (for entropy) have? I am curious to learn what problem this PR fixes.

Reply:
This changes the randomness to match recent changes in https://github.com/filecoin-project/specs-actors/commit/bf4c30853a074d1d3718a7d814da6ac700a20828